### PR TITLE
Update Integration Tests

### PR DESCRIPTION
**Pull request by chris-rock**
- add windows support for yml, csv and ini integration tests
- add ec2 example

### CODE REVIEW

File: test/integration/test/integration/default/file_spec.rb [hunk 1]
```diff
@@ -106,4 +105,9 @@
     its('type') { should eq :directory }
   end
 
+elsif os.windows?
+  describe file('C:\\Windows') do
+    it { should exist }
```
Comment (chris-rock) [inline test/integration/test/integration/default/file_spec.rb:137]: adds a test for #288

**Comment by chris-rock** [comment]
TODO: Add integration test for security policy, see #321

File: test/integration/.kitchen.ec2.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
+driver:
+  name: ec2
+  region: eu-central-1
+  instance_type: m3.large
+  associate_public_ip: true
+  interface: dns
+
+transport:
+  ssh_key: /Users/chartmann/Development/key/aws_chartmann.pem
```
Comment (arlimus) [inline test/integration/.kitchen.ec2.yml]: Let's fix this in the next MR.

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.getchef.com/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## GitHub Users Who Are Not Authorized To Contribute

The following GitHub users do not appear to have signed a CLA:

* @arlimus

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

**Comment by arlimus** [comment]
Fantastic new integration tests, kudos @chris-rock :+1:

**Pull request merged**